com.unity.render-pipelines.high-definition/Runtime/Core/Migration/MigrationDescription.cs

```diff
@@ -104,10 +104,16 @@ public bool Migrate(TTarget target)
             }
 
 #if UNITY_EDITOR
-            // Special in prefab case
-            if (target is UnityEngine.Object && UnityEditor.PrefabUtility.IsPartOfNonAssetPrefabInstance(target as UnityEngine.Object))
+            UnityEngine.Object targetObject = target as UnityEngine.Object;
+            if (targetObject != null)
```

Comment:
I believe this should never happens but yes better check this. 👍

Reply:
same, the code was already checking for it and i was on the safe side :)